sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/models/ContainerRepositoryProperties.java

```diff
@@ -15,7 +15,7 @@
 
 /** Properties of this repository. */
 @Fluent
-public class ContainerRepositoryProperties implements JsonSerializable<ContainerRepositoryProperties> {
+public final class ContainerRepositoryProperties implements JsonSerializable<ContainerRepositoryProperties> {
```

Comment:
Isn't marking a class as `final` technically a breaking change? Ignore this comment if this class has not been made GA before.

Reply:
Good catch! It has bee final in the prev GA version, but final got lost since then.